pkg/controller/admissionchecks/multikueue/batchjobadaptor.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package multikueue
+
+import (
+	"context"
+
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"sigs.k8s.io/kueue/pkg/controller/constants"
+)
+
+type batchJobAdaptor struct{}
+
+var _ jobAdaptor = (*batchJobAdaptor)(nil)
+
+var (
+	// The list of labels that need to be removed before creating the job in the worker cluster.
+	cleanupLabels = []string{"job-name", "controller-uid", "batch.kubernetes.io/job-name",
```

Comment:
This is the same set of labels as here: https://github.com/kubernetes-sigs/kueue/blob/2ee9e0f6eea07e25be6722c314dbb9a7c9a79ca1/pkg/controller/jobs/job/job_controller.go#L214. I would prefer to commonize them